src/index.html

```diff
@@ -18,7 +18,7 @@ <h1 class="tittle">To Do List</h1>
     <section class="create-to">
       <form id="form-todo">
         <input type="text" name="content" id="content" placeholder="Add to your list..." aria-label="content" required>
-        <button type="submit"><img id="arrowimg" src="./image/arrow.png" alt="Add task - arrow icon" ></button>
+        <button type="submit"><img id="arrowimg" src="image/arrow.png" alt="Add task - arrow icon" ></button>
       </form>
     </section>
     <section class="todo-li">
```

Comment:
- [ ] [OPTIONAL] It would be nice if you implement that `HTML` codes on `DOM` like your tasks `Array`  👍🏼

- `[HINT]`
```javascript
const populateTasks = (task, parentElement) => {
  const { description, completed, index: id } = task;
  const innerHTML = `
         <p>${description}</p>
         <p>${completed ? 'completed' : 'not completed'}</p>`;
  const newTodoListElement = document.createElement('li');
  Object.assign(newTodoListElement, { innerHTML, id });
  parentElement.appendChild(newTodoListElement);
};
```